documentation/modules/cruise-control/proc-approving-optimization-proposal.adoc

```diff
@@ -34,48 +34,53 @@ To do so, refresh the optimization proposal to make sure it uses the latest clus
 +
 [source,shell,subs="+quotes"]
 ----
-kubectl annotate kafkarebalance _rebalance-cr-name_ strimzi.io/rebalance=refresh
+kubectl annotate kafkarebalance _<kafka_rebalance_resource_name>_ strimzi.io/rebalance=refresh
 ----
 
-.. Check the status of the `KafkaRebalance` resource:
+. Check the status of the optimization proposal:
 +
 [source,shell,subs="+quotes"]
 ----
-kubectl describe kafkarebalance _rebalance-cr-name_
+kubectl get kafkarebalance -o wide -w -n _<namespace>_
```

Comment:
The `-w` watches the resource. So with that, I would maybe expect more something like _Wait for the optimization proposal to move into the state ?????_ on line 40. Or remove the `-w` from the comment which would more correspond to the _Check ..._ as a one of thing.

Reply:
Changed line 40 to:

> Wait for the status of the optimization proposal to change to `ProposalReady`: